framework/scripts/wazuh_server.py

```diff
@@ -503,7 +503,7 @@ async def monitor_server_daemons(loop: asyncio.BaseEventLoop, server_process: ps
     """
     comms_api_config = CentralizedConfig.get_comms_api_config()
     process_children = {
-        MANAGEMENT_API_DAEMON_NAME[:15]: 3,
+        MANAGEMENT_API_DAEMON_NAME[:15]: 5,
```

Comment:
This is one process from each pool (process_pool, authentication_pool, events_pool) plus one for each manager (commands, RBAC)